src/sentry_plugins/github/endpoints/webhook.py

```diff
@@ -310,10 +310,28 @@ def _handle_created(self, event, organization, repo, is_apps):
         user = pull_request['user']
         merge_commit_sha = pull_request['merge_commit_sha']
 
-        # get author TODO(Maxbittker): do we need to use the more complicated
-        # logic from the commit webhook?
-        author = author = CommitAuthor.objects.get_or_create(
+        author_email = u'{}@localhost'.format(user['login'])
```

Comment:
mmm why localhost and not trying to match the github noreply emails?

Reply:
emulating https://github.com/getsentry/sentry-plugins/blob/fix/pr/commit_author/src/sentry_plugins/github/endpoints/webhook.py#L130-L132
(and what's in the db already)